csharp/ql/src/utils/model-generator/CaptureSummaryModels.qll

```diff
@@ -1,21 +1,4 @@
-/**
- * @name Capture summary models.
- * @description Finds applicable summary models to be used by other queries.
- * @id java/utils/model-generator/summary-models
- */
-
-import java
-import semmle.code.java.dataflow.TaintTracking
-import semmle.code.java.dataflow.internal.DataFlowImplCommon
-import semmle.code.java.dataflow.internal.DataFlowNodes
-import semmle.code.java.dataflow.internal.DataFlowPrivate
-import semmle.code.java.dataflow.InstanceAccess
-import ModelGeneratorUtils
-
-string captureFlow(TargetAPI api) {
-  result = captureQualifierFlow(api) or
-  result = captureThroughFlow(api)
-}
+import CaptureSummaryModelsSpecific
```

Comment:
private import

Reply:
We need to re-expose the TargetAPI class, which is defined in CaptureSummaryModelsSpecific.